chart/kubeapps/values.yaml

```diff
@@ -22,18 +22,18 @@ ingress:
   # If certManager is set to true, annotation kubernetes.io/tls-acme: "true" will automatically be set
   annotations:
   #  kubernetes.io/ingress.class: nginx
-    
+
   # The list of hostnames to be covered with this ingress record.
   # Most likely this will be just one host, but in the event more hosts are needed, this is an array
   hosts:
-  - name: kubeapps.local
-    path: /
+    - name: kubeapps.local
```

Comment:
did your editor change this automatically or something? I would prefer if we didn't change the indentation.

Reply:
yes... It's really bothering me when it changes YAMLs